kustomizations/apps/data-hub-configs/bigquery-to-opensearch--stg.yaml

```diff
@@ -16,24 +16,21 @@ bigQueryToOpenSearch:
               ) AS authors,
               s2_response.embedding.vector AS s2_specter_embedding_v1,
               s2_response.tldr,
-              s2_response.provenance.imported_timestamp AS data_hub_imported_timestamp,
-              s2_response.provenance.request_timestamp AS data_hub_request_timestamp,
-              s2_response.provenance.response_timestamp AS data_hub_response_timestamp,
-              s2_response.paperId AS s2_paper_id,
-              europepmc_response.firstPublicationDate AS publication_date
+              s2_response.provenance.imported_timestamp AS s2_data_hub_imported_timestamp,
+              s2_response.provenance.request_timestamp AS s2_data_hub_request_timestamp,
+              s2_response.provenance.response_timestamp AS s2_data_hub_response_timestamp,
+              s2_response.paperId AS s2_paper_id
           FROM `elife-data-pipeline.{ENV}.v_latest_semantic_scholar_response` AS s2_response
-          LEFT JOIN `elife-data-pipeline.{ENV}.v_latest_europepmc_preprint_servers_response` AS europepmc_response
-              ON europepmc_response.doi = s2_response.externalIds.DOI
           WHERE s2_response.externalIds.DOI IS NOT NULL
     fieldNamesFor:
       id: doi
-      timestamp: data_hub_imported_timestamp
+      timestamp: s2_data_hub_imported_timestamp
     state:
       initialState:
         startTimestamp: '2022-05-01+00:00'
       stateFile:
         bucketName: '{ENV}-elife-data-pipeline'
-        objectName: 'airflow-config/bigquery-to-opensearch/{ENV}-state.json'
+        objectName: 'airflow-config/bigquery-to-opensearch/{ENV}-state-s2.json'
```

Comment:
I changed the state filename. It will be useful to copy the previous state file before merging to avoid having to process all of the Semantic Scholar data again.